findLongest.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * This method is to find the longest element in a Array
+ *
+ * @since 5.0.0
+ * @category Array
+ * @param {Array} collection The array to inspect.
+ * @returns {string} Returns  longest element in a Array
+ * @example
+ *
+ * findLongest(["abc", 'abcd', "abcde"])
+ * // => "abcde"
+ */
+function findLongest(array) {
+  if (!array.length) return "";
```

Comment:
Put a check a head of it to check whether the 'array' param is an array or not and if its not an array then throw an exception.